Dockerfile

```diff
@@ -22,6 +22,13 @@ RUN cd /app && npm install
 COPY test/Gemfile test/Gemfile.lock /app/test/
 RUN cd /app/test && gem install bundler && bundle install --jobs 20 --retry 5
 
+# Sauce Labs proxy
+RUN mkdir -p /app/test/spec/sauce
+RUN cd /app/test/spec/sauce && curl -O https://saucelabs.com/downloads/sc-4.4.2-linux.tar.gz
+# TODO: throw error if `openssl sha1 sc-4.4.2-linux.tar.gz` != '57a07a14c5d95d72b6606ba34fceaf5bf76c2865'
+RUN pwd
+RUN cd /app/test/spec/sauce && tar -zxvf sc-4.4.2-linux.tar.gz
+
```

Comment:
I thought this would work but it didn't seem to last I tried, and I had to run this commands manually inside the container.

I'm not sure what the best location is – it doesn't need to be accessible from outside the container (and needs to be inside to work), so maybe a directory other than /app...

@bfoote if you know how to do line 28 that would be helpful :)